package.json

```diff
@@ -55,6 +55,8 @@
     "@keplr-wallet/wc-qrcode-modal": "0.12.23",
     "@reduxjs/toolkit": "1.9.7",
     "@sentry/react": "7.81.0",
+    "@stripe/react-stripe-js": "^2.4.0",
```

Comment:
nit: preferrably exact version, as it's easy to see which we are running without looking at the lock file.